suitcase/tiff/tests.py

```diff
@@ -1,2 +1,67 @@
-# Tests should generate (and then clean up) any files they need for testing. No
-# binary files should be included in the repository.
+from . import export
+import numpy
+from numpy.testing import assert_array_equal
+import pytest
+from suitcase.utils.conftest import (simple_plan,
+                                     multi_stream_one_descriptor_plan,
+                                     one_stream_multi_descriptors_plan)
+import tempfile
+import tifffile
+
+@pytest.fixture()
+def expected():
+    expected_array = numpy.ones((10, 10))
+
+    return expected_array
+
+@pytest.fixture(params=[True, False], scope='function')
+def stack_images(request):
+    return request.param
+
+
+def test_simple_plan(events_data, expected, stack_images):
+    ''' runs a test using a simple count plan with num=5
+    '''
+    run_plan_test(simple_plan, events_data, expected, stack_images)
+
+
+def test_multi_stream_one_descriptor_plan(events_data, expected, stack_images):
+    ''' runs a test using a simple count plan with num=5
+    '''
+    run_plan_test(multi_stream_one_descriptor_plan, events_data, expected,
+                  stack_images)
+
+
+def test_one_stream_multi_descriptors_plan(events_data, expected, stack_images):
+    ''' runs a test using a simple count plan with num=5
+    '''
+    run_plan_test(one_stream_multi_descriptors_plan, events_data, expected,
+                  stack_images)
+
+
+def run_plan_test(plan, events_data, expected, stack_images):
```

Comment:
Given that each of the tests in this file run this function, I would expect it to be spelled

```python
@pytest.mark.parametrize('plan', [simple_plan, multi_stream_one_descriptor_plan, one_stream_multi_descriptors_plan])
@pytest.mark.parametrize('stack_images', [True, False])
def run_plan_test(plan, events_data, expected, stack_images):
    ...
```

Reply:
Can you parametrize functions that aren't tests? If this works, strongly agree it is better, but not sure it works.

Reply:
actually @danielballan is correct, but I have now just made `run_plan_test` the test (with a different name) and removes the tests above. This was a great way to make the test fiel neater, thanks @tacaswell.

Reply:
Sorry for missing that you had to rename the function to make it be found as a test.